Josh cook3/issue186 new text on non demographic growth for ndg3 (#195)
The-Strategy-Unit/nhp_project_information: modelling_methodology/non-demographic_growth.qmd

---
title: "Non-Demographic Growth"
order: 4
--- 
 

> When considering growth in healthcare activity, it is standard practice to distinguish between growth that can be directly attributed to changes in the size and makeup of the population (demographic growth) and growth that is driven by other factors (which is typically grouped together under the heading of 'non-demographic growth'). When looking back, this non-demographic growth (NDG) is the residual between observed overall growth and the element of that that can be explained by demographic change.

Reviewer: ```suggestion
When considering growth in healthcare activity, it is standard practice to distinguish between growth that can be directly attributed to changes in the size and makeup of the population (demographic growth) and growth that is driven by other factors (which is typically grouped together under the heading of 'non-demographic growth'). When looking back, this non-demographic growth (NDG) is the residual (difference) between observed overall growth and the element of that which can be explained by demographic change.
```

"Residual" is a technical term. added in alternative synonym to aid understanding.
Also used another synonym to get rid of double "that"

| Activity Type             | Primary Rate<br>Variant 2<br>(P10/P90) | Sensitivity<br>Variant 3<br>(P10/P90) |
| ------------------------- | -------------------------------------- | ------------------------------------- |
| A&E - ambulance           | 1.31%<br>(1.21%/1.41%)                 | 1.49%<br>(-0.54%/3.52%)               |
| A&E - walk-in             | 1.31%<br>(1.21%/1.41%)                 | 1.49%<br>(-0.54%/3.52%)               |
| Inpatients - elective     | 0.62%<br>(0.53%/0.71%)                 | 1.35%<br>(-0.01%/2.71%)               |
| Inpatients - maternity    | 0%                                     | 0%                                    |
| Inpatients - non-elective | -0.14%<br>(-0.45%/0.17%)               | 1.16%<br>(-0.37%/2.68%)               |
| Outpatients - first       | 2.49%<br>(2.28%/2.7%)                  | 2.45%<br>(0.02%/4.87%)                |
| Outpatients - follow-up   | 2.49%<br>(2.28%/2.7%)                  | 2.45%<br>(0.02%/4.87%)                |
| Outpatients - procedure   | 2.49%<br>(2.28%/2.7%)                  | 2.45%<br>(0.02%/4.87%)                |

> NB NDG adjustments are not applied to length of stay. 

Reviewer: ```suggestion
Note that NDG adjustments are not applied to length of stay. 
```
NB is Latin abbreviation and not user-friendly

Currently NDG adjustments are not applied for inpatients maternity activity. The parameter is set at 0% annual growth rate which represents no change. This assumption is currently under review and may be revised in later versions of the model.

The table in the inputs app shows the NDG adjustment rates as the multiplier values rather than the annual percentage growth rates e.g. 1.0121 instead of 1.21%. 

Note that the adjustment rates are distributions - they range between a P10 and a P90 value which are shown within brackets in the table above. For more information about how the model simulations randomly samples a single value from these distributions, please see the [worked example](https://connect.strategyunitwm.nhs.uk/nhp/project_information/modelling_methodology/modelling_uncertainty.html#a-worked-example). 

> Further work is planned to seperate out some of the key components of non-demographic growth such as the impact of new medical technologies, changes in patient expectations, and changes in clinical standards. 

Reviewer: ```suggestion
Further work is planned to seperate out some of the key components of non-demographic growth such as the impact of new medical technologies, changes in patient expectations, and changes in clinical standards. 

To see a detailed note describing the adoption of Variant 2 as the model's primary rate for non-elective inpatient admissions, please [see here](https://github.com/The-Strategy-Unit/nhp_project_information/blob/77df9d50923e5242e95226f56112d90d1ee562bf/modelling_methodology/non-demographic_growth.qmd).
```

| Outpatients - follow-up   | 2.49%<br>(2.28%/2.7%)                  | 2.45%<br>(0.02%/4.87%)                |
| Outpatients - procedure   | 2.49%<br>(2.28%/2.7%)                  | 2.45%<br>(0.02%/4.87%)                |

Note that NDG adjustments are not applied to length of stay. 

Currently NDG adjustments are not applied for inpatients maternity activity. The parameter is set at 0% annual growth rate which represents no change. This assumption is currently under review and may be revised in later versions of the model.

The table in the inputs app shows the NDG adjustment rates as the multiplier values rather than the annual percentage growth rates e.g. 1.0121 instead of 1.21%. 

Note that the adjustment rates are distributions - they range between a P10 and a P90 value which are shown within brackets in the table above. For more information about how the model simulations randomly samples a single value from these distributions, please see the [worked example](https://connect.strategyunitwm.nhs.uk/nhp/project_information/modelling_methodology/modelling_uncertainty.html#a-worked-example). 

> Further work is planned to seperate out some of the key components of non-demographic growth such as the impact of new medical technologies, changes in patient expectations, and changes in clinical standards. 

Reviewer 2: ```suggestion
Further work is planned to separate out some of the key components of non-demographic growth such as the impact of new medical technologies, changes in patient expectations, and changes in clinical standards. 
```
